test/e2e/rte/rte.go

```diff
@@ -71,9 +71,9 @@ var _ = ginkgo.Describe("[RTE][InfraConsuming] Resource topology exporter", func
 	ginkgo.BeforeEach(func() {
 		var err error
 
-		err = e2etestns.Setup(f)
+		nsCleanup, err := fixture.F.CreateNamespace("rte")
```

Comment:
I kinda like we seem to want to mimic `testing.T` with `fixture.F`, but I don't like too much relying on a global variable to do this work. It would be nicer if we create our own `fixture.F` on each `BeforeEach` block

Reply:
Initializing the `Fixture` for each case seems a bit expensive don't you think?
It essentially mean we need to get the rest config and initialize all the client again and again for each test.
What benefit do we gain here?

BTW the line   nsCleanup, err := fixture.F.CreateNamespace("rte") changed a bit in later commit (mostly cosmetics).

Reply:
I think we should (strongly) prioritize isolation over speed. Re-initializing clients every time is wasteful indeed and we can probably avoid that using a package to holds clients. We can initialize them explicitely and/or with `init`. This is close to what https://github.com/openshift-kni/numaresources-operator/blob/main/test/utils/clients/clients.go does